src/os/freertos/include/AbstractTask.tpp

```diff
@@ -5,8 +5,10 @@ template <unsigned int stackSize>
 AbstractTask<stackSize>::AbstractTask(const char* taskName, UBaseType_t priority) {
     m_taskName = taskName;
     m_priority = priority;
-    m_taskHandle = NULL;
     m_taskBuffer = {};
+
```

Comment:
Task created/started on construction of the AbstractTask? Is this necessary? I suspect this change might break a lot of things